src/cli/argument-parser.js

```diff
@@ -203,8 +203,11 @@ export default class CLIArgumentParser {
     }
 
     async _parseVideoOptions () {
-        this.opts.videoOptions         = typeof this.opts.videoOptions === 'string' ? await getVideoOptions(this.opts.videoOptions) : null;
-        this.opts.videoEncodingOptions = typeof this.opts.videoEncodingOptions === 'string' ? await getVideoOptions(this.opts.videoEncodingOptions) : null;
+        if (this.opts.videoOptions)
+            this.opts.videoOptions = await getVideoOptions(this.opts.videoOptions);
+
+        if (this.opts.videoEncodingOptions)
```

Comment:
Raise an error if video encoding options were specified without video options.

Reply:
It's already done: https://github.com/DevExpress/testcafe/blob/master/src/runner/index.js#L241